automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -185,6 +189,16 @@ def product_upgrade(product):
                     execute(satellite6_upgrade, host=sat_host)
                 else:
                     execute(satellite6_zstream_upgrade, host=sat_host)
+                # Execute following tasks post upgrade
```

Comment:
I think it is better to create one function inside file 'automation_tools/satellite6/upgrade/tasks.py' with name something like 'pre_test_tasks'. And call that task here !

Otherwise this product_upgrade will keep on increasing.